root/layout/components/sidebar/InstrumentSidebar.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {withCatalystContext} from '../../../context';
+import CommonsImage from '../../../static/scripts/common/components/CommonsImage';
+import {l} from '../../../static/scripts/common/i18n';
+import entityHref from '../../../static/scripts/common/utility/entityHref';
+import ExternalLinks from '../ExternalLinks';
+
+import AnnotationLinks from './AnnotationLinks';
+import CollectionLinks from './CollectionLinks';
+import EditLinks from './EditLinks';
+import LastUpdated from './LastUpdated';
+import MergeLink from './MergeLink';
+import RemoveLink from './RemoveLink';
+import SidebarLicenses from './SidebarLicenses';
+import {SidebarProperty, SidebarProperties} from './SidebarProperties';
+import SidebarTags from './SidebarTags';
+import SidebarType from './SidebarType';
+import SubscriptionLinks from './SubscriptionLinks';
+
+type Props = {|
+  +$c: CatalystContextT,
+  +instrument: InstrumentT,
+|};
+
+const InstrumentSidebar = ({$c, instrument}: Props) => {
+  return (
+    <div id="sidebar">
+      <CommonsImage
```

Comment:
This does load the Commons images - not sure if that's blocked elsewhere in prod, but we're supposed not to show them for now :)

Reply:
Yeah, that's controlled by a DBDefs flag set in prod. You should still see them locally if you haven't set that flag.